app/uk/gov/hmrc/tariffclassificationfrontend/views/reject_case.scala.html

```diff
@@ -19,23 +19,33 @@
 @import uk.gov.hmrc.tariffclassificationfrontend.models.Case
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.contact_details
+@import uk.gov.hmrc.tariffclassificationfrontend.views.forms.components.RadioOption
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_radiogroup
 
-@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(c: Case, form: Form[Boolean])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @main(bodyClasses = None) {
 
     @case_heading(c)
 
     <div class="grid-row">
         <div class="column-full">
-            <h2 class="heading-large mt-0">Reject this case</h2>
+            <h2 class="heading-large mt-0">Contact the applicant and attach a copy of the email before you reject this case</h2>
 
-            <p> Before you reject this case, add a note explaining why you’re rejecting it. </p>
-            <p>The days elapsed count will stop at <span id="days-elapsed">@(c.daysElapsed)</span>.</p>
+            @contact_details(c)
+
+            <h3 class="heading-medium mt-0">Have you contacted the applicant and attached the email to this case?</h3>
 
             @helper.form(action = RejectCaseController.confirmRejectCase(c.reference), 'id -> "reject-case") {
                 @helper.CSRF.formField
 
+                @input_radiogroup(
```

Comment:
Do you think there's any value in a common `input_yes_no` component?

Reply:
Yes. If I have time I will try and extract one

Reply:
Done